src/server/survey_and_program_data.R

```diff
@@ -2,27 +2,28 @@ library(magrittr)
 
 getProgramDataInWideFormat <- function(country) {
     long <- prgm_dat[prgm_dat$country == country, ]
-    long$country <- NULL
+    long$country <- as.character(long$country)
```

Comment:
Have reintroduced `country` to the table, so that we can do basic validation of any uploaded data.